documentation/latest/intro/usecases.md

```diff
@@ -24,3 +26,11 @@ low-bandwidth connection (GPRS, LoraWAN, NB-IoT) will work without issues.
 IA on the edge is a has tons of posibilities, Flotta device agent is available
 on ARM or x86, also has support for any kind of Nvidia Jetson boards, the main
 boards for IA in the edge.
+
+## Industry
+
+When a factory with thousands of different sensors, robotics arms, and multiple
+computers that needs to run each one a different workload, flotta provides a way
+to manage all these devices from a single user place.
+
+![](/assets/images/manufactor_line.png)
```

Comment:
the image contains a typo, should be `manufacture line` (instead of manufacter line)

also the name of the png file can be updated as well.

and last, why is the image tilted?